tests/exit-status.sh

```diff
@@ -1,43 +1,58 @@
 #!/usr/bin/env bash
-# Grab the execution directory
-SCRIPT_DIR="$( dirname "$( readlink -f "${BASH_SOURCE[0]}" )")"
+
+# Grab the execution directory.
+SCRIPT_FILE="$( readlink -f "${BASH_SOURCE[0]}" 2> /dev/null )"
+if [[ $? ]]; then
+  SCRIPT_FILE="$( readlink "${BASH_SOURCE[0]}" )"
+fi
+SCRIPT_DIR="$( dirname "$SCRIPT_FILE" )"
 export SCRIPT_DIR
 
-cd /tmp || exit
-if  [[ ! -e "./veraPDF-corpus-master" ]]
-then
+if (( NO_CD != 1 )); then
+  cd /tmp || exit
+fi
+
+if [[ ! -e "./veraPDF-corpus-master" ]]; then
   wget https://github.com/veraPDF/veraPDF-corpus/archive/master.zip
   unzip master.zip
   rm master.zip
 fi
-./verapdf/verapdf "./veraPDF-corpus-master/PDF_A-1b/6.1 File structure/6.1.4 Cross reference table/veraPDF test suite 6-1-4-t03-pass-b.pdf"
+
+if [[ -z "$VERAPDF" ]]; then
+  VERAPDF_PATH=./verapdf/verapdf
```

Comment:
[Codacy points out that this isn't used](https://app.codacy.com/gh/veraPDF/veraPDF-apps/pullRequest?prid=8443762) and again lines 21-23 can probably be removed as well.

Reply:
Sorry, line 22 should be assigning `VERAPDF`, that was my mistake. I set this as an environment variable in the install script for the package, since it's not located at `./verapdf/verapdf` in that case. Hope that's alright.